routes/scratch.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Scratch Routes
+|--------------------------------------------------------------------------
+|
+| This is where we store all routes that are no longer actively used within
+| the application but want to keep around for debugging / local dev purposes.
+|
+*/
+
+// Password Reset routes:
+Route::get( 'password/reset','Auth\ForgotPasswordController@showLinkRequestForm')->name('password.request');
+Route::post('password/email','Auth\ForgotPasswordController@sendResetLinkEmail') ->name('password.email');
+Route::get( 'password/reset/{token}','Auth\ResetPasswordController@showResetForm')->name('password.reset');
+Route::post('password/reset','Auth\ResetPasswordController@reset');
+
+// Email Validation routes
+Route::get('email/verify','Auth\VerificationController@show')->name('verification.notice');
+Route::get('email/verify/{id}','Auth\VerificationController@verify')->name('verification.verify');
+Route::get('email/resend','Auth\VerificationController@resend')->name('verification.resend');
+
+ Route::group(['prefix' => 'item'], function () {
+     Route::get( '/{item_id}/{slug?}', 'ItemController@show')->name('item.show');
+ });
+
+Route::get( '/about','HomeController@about')->name('about');
+Route::get( '/contact','HomeController@contact')->name('contact');
```

Comment:
Please comment these out. They point to placeholder pages and aren't ready to be live.

Reply:
These are in a file that is not included in case that makes a difference, if not I will comment :)

Reply:
Gotcha.